### ISSUE TITLE
Add to existing App

### ISSUE THREAD

**Issue by sGerli** [angular/mobile-toolkit#151]
Is there a way to add this to an existing Angular app made with the cli?
And if there is I think it's a good idea to document it

**Comment by webmaxru** [comment]
Yes. Just follow this doc: https://bit.ly/pwa-angularsummit-2017
